solidity/solc-update-0.8.15-to-0.8.25.md

```diff
@@ -0,0 +1,98 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Solidity Version Upgrade Proposal](#solidity-version-upgrade-proposal)
+  - [Benefits to the OP Stack](#benefits-to-the-op-stack)
+      - [DevX](#devx)
+      - [Gas Optimization](#gas-optimization)
+      - [Features](#features)
+  - [Notable Features](#notable-features)
+  - [Notable Bug Fixes](#notable-bug-fixes)
+  - [Changelog Links](#changelog-links)
+  - [Action Items](#action-items)
+  - [Additional Notes](#additional-notes)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                     |                |
+| ------------------- | -------------- |
```

Comment:
If we like this table (which I do), let's also add it to the template

Reply:
@smartcontracts